packages/builder/src/components/start/CreateAppModal.svelte

```diff
@@ -159,6 +162,7 @@
     />
   {/if}
   <Input
+    autofocus={true}
```

Comment:
NAB: just an FYI - but this is equivalent to just `autofocus`. You can skip the `={true}` part when just passing in a boolean prop like this.

Reply:
In this instance, there is no variable in the parent component called 'autofocus'. If I don't explicitly set the value to 'true', it'll end up undefined in the Input component

Reply:
Just passing `autofocus` is the same as `autofocus={true}` - I think you're thinking of `{autofocus}` (wrapped in curlies) which passes in the parent value like you say. But if you remove the curlies then the behaviour is different :+1:

Reply:
You don't need to change this or anything lol but it's just a tip for the future!